var/spack/repos/builtin/packages/mfem/package.py

```diff
@@ -50,6 +50,10 @@ class Mfem(Package):
     # If this quick verification procedure fails, additional discussion
     # will be required to verify the new version.
 
+    version('3.3',
+            'b17bd452593aada93dc0fee748fcfbbf4f04ce3e7d77fdd0341cc9103bcacd0b',
+            url='http://goo.gl/Vrpsns', extension='.tgz')
```

Comment:
All 3 versions have the file extension. As far as I know, Spack only uses `extension` to decide how to rename the file it downloads. Spack will rename `Vrpsns` to `mfem-3.3.tgz`. Since `.tar.gz` and `.tgz` are the same thing, it doesn't really matter which we use. I'm just OCD and like consistency 😆